portfolio/src/main/webapp/script.js

```diff
@@ -46,3 +46,29 @@ function addRandomQuote() {
       delay: (el, i) => 70*i
     })
 }
+
+/**
+ * Sends an email when someone fills out the contact me page. 
+ */
+function contactEmail() {
+    // Obtain name, email, and message from the contact page.
+    const formData = {
+        'name': $('input[name=name]').val(),
+        'email': $('input[name=email]').val(),
+        'message': $('textarea[name=message]').val()
+    };
+    console.log(formData);
+
+    const subject = formData.name + " - " + formData.email;
+
+    //Send an email containg the name and email in the subject line and the message in the body.
+    Email.send({
+    Host : "smtp.gmail.com",
+    Username : "betterbnbcontact@gmail.com",
+    Password : "betterBNB1234",
```

Comment:
Not good to store username and password in plain text... (esp on public repo)... Find a way to get an obfuscated key? There seem to be some APIs, and also in the next section (next week) you will be implementing some functionalities server-side, which this email sending workflow can be a part of too.

Reply:
I will make a new email address and find some API to encrypt the credentials. Yeah, I think it will be better to add the email sending functionality server-side. I will do that after I get the server set up next week.